packages/cozy-konnector-libs/src/libs/saveFiles.js

```diff
@@ -519,3 +644,7 @@ function getValOrFnResult(val, ...args) {
     return val.apply(val, args)
   } else return val
 }
+
+function calculateFileKey(entry, filePrimaryKeys) {
+  return filePrimaryKeys.map(key => get(entry, key)).join('####')
```

Comment:
We joining all attributes here, as discuss, we confirmed that a futur change in a connector code, on the sending of file by differents pieces of code could resume in a different order.

Need a sort to be sure to keep always the same order

Reply:
Sort added in next commit